OpenGL/Math/Vector4.cs

```diff
@@ -342,6 +342,39 @@ public static void Swap(ref Vector4 v1, ref Vector4 v2)
             v1 = v2;
             v2 = t;
         }
+
+        public float this[int a]
```

Comment:
I think we should probably deprecate this type of getter/setter, since System.Numerics doesn't support it.